docs/source/docker/getting_started/first_steps.rst

```diff
@@ -462,34 +444,7 @@ an automatically generated PDF report.
 Sample reports can be found on the `Lightly website <https://lightly.ai/analytics>`_.
 
 
-.. _ref-docker-runs:
-
-Live View of Docker Status
```

Comment:
Where did we move this section to? I think it's still useful to have the option of a "live view"

Reply:
It's explained under "Scheduling a Simple Job"